external/openmeeg/testOpenMEEGeeg.m

```diff
@@ -5,16 +5,16 @@
 
 % Copyright (C) 2010-2017, OpenMEEG developers
 
-addpath(cd) % Make sure current folder is in the path
+addpath(pwd) % Make sure current folder is in the path
 
 %% Set the position of the probe dipole
 dippos = [0 0 70];
 
 %% Set the radius and conductivities of each of the compartments
 
 % 4 Layers
-r = [85 88 92 100];
-c = [1 1/20 1/80 1];
+r = [100 92 88 85];
+c = [1 1/80 1/20 1];
```

Comment:
why did you need this?

Reply:
"Our pipeline expects outside->inside order of the surfaces, so I have changed those accordingly." :-)

Reply:
well I think the right fix is to unsure that your pipeline is smart enough to support previously documented code rather than changing the tests :

Reply:
Documented where? The previous code calls a function that tries to rearrange code from outside to inside as well, it happens to also fail at it. Other fieldtrip functions and leadfield_openmeeg.m are also smart enough to rearrange when given anatomical labels (e.g. "scalp", "skull", "brain").

Reply:
well I don't fully see the impact of this change, it just seems inappropriate to change the test to make it pass. test should ensure there is no regression so I prefer to change the functions so the old tests still pass as is.

Reply:
Well if every implementation expects outside to inside, or alternatively labels to indicate what each layer corresponds to, then it seems that it's the test that should change. No regression is ensured since the output values match exactly, plus this is what is used in practice for real analyses.